src/config.ts

```diff
@@ -17,7 +17,7 @@ type DeepPartial<T> = {
 // interface this library uses internally
 export interface SnowpackConfig {
   source: 'local' | 'pika';
-  webDependencies?: string[];
+  webDependencies?: string[] | {[packageName: string]: string};
```

Comment:
💯! TypeScript is so nice. It’s hard to remember what supporting different data structures for backwards compatibility was like before typed JS